src/controlCenter/widgets/backlight/backlight.vala

```diff
@@ -0,0 +1,73 @@
+using GLib;
+
+namespace SwayNotificationCenter.Widgets {
+    public class Backlight : BaseWidget {
+        public override string widget_name {
+            get {
+                return "backlight";
+            }
+        }
+
+        BacklightUtil client;
+
+        Gtk.Label label_widget = new Gtk.Label (null);
+        Gtk.Scale slider = new Gtk.Scale.with_range (Gtk.Orientation.HORIZONTAL, 0, 100, 1);
+
+        public Backlight (string suffix, SwayncDaemon swaync_daemon, NotiDaemon noti_daemon) {
+            base (suffix, swaync_daemon, noti_daemon);
+
+            Json.Object ? config = get_config (this);
+            if (config != null) {
+                string ? label = get_prop<string> (config, "label");
+                label_widget.set_label (label ?? "Brightness");
+                string device = (get_prop<string> (config, "device") ?? "intel_backlight");
+                string subsystem = (get_prop<string> (config, "subsystem") ?? "backlight");
+                int min = get_prop<int> (config, "min");
```

Comment:
Adding a int.min() to would ensure that it won't go under 0: `int min = int.max(0, get_prop<int>(config, "min"));`